app/src/main/java/com/parking_system_kotlin/mvp/presenter/ParkingPresenter.kt

```diff
@@ -0,0 +1,18 @@
+package com.parking_system_kotlin.presenter
+
+import com.parkingsystemkotlin.listeners.ListenerSetParkingDialogFragment
+import com.parkingsystemkotlin.mvp.contracts.ParkingContract
+
+class ParkingPresenter(model: ParkingContract.MainActivityModel, view: ParkingContract.MainActivityView) : ParkingContract.MainActivityPresenter{
+    private var model: ParkingContract.MainActivityModel = model
+    private var view: ParkingContract.MainActivityView = view
```

Comment:
if you are using the koltin constructor you could use this parameters as variables.
Improve your code using :
class ParkingPresenter(**private val** model: ParkingContract.MainActivityModel, **private val** view: ParkingContract.MainActivityView)

Reply:
nice!. done